compiler/scanner.mll

```diff
@@ -5,7 +5,7 @@
  * COMS4115
  *)
 
-{ open Parser }
```

Comment:
Yep and we'll change it back when we have a parser. But we can't run the Scanner for testing with an empty parser, which is why we have the dummy token file hooked up now.